core/common/src/main/java/alluxio/ClientContext.java

```diff
@@ -123,8 +124,11 @@ public synchronized void loadConf(InetSocketAddress address, boolean loadCluster
     GetConfigurationPResponse response = Configuration.loadConfiguration(address,
         conf, !loadClusterConf, !loadPathConf);
     if (loadClusterConf) {
-      mClusterConf = Configuration.getClusterConf(response, conf, Scope.CLIENT);
+      InstancedConfiguration clusterConf =
+          Configuration.getClusterConf(response, conf, Scope.CLIENT);
```

Comment:
The `Scope.CLIENT` will limit the scope of configuration to that of clients' only. If we want to make sure workers can benefit from this feature then this is not sufficient.

Reply:
Yes, I can fix it.